pydoctor/test/test_epydoc2stan.py

```diff
@@ -38,6 +40,62 @@ class E:
     epydoc2stan.format_docstring(mod.contents['D'])
     epydoc2stan.format_docstring(mod.contents['E'])
 
+def test_func_arg_and_ret_annotation():
+    annotation_mod = fromText('''
+    def f(a: List[str]) -> bool:
+        """
+        @param a: an arg, a the best of args
+        @return: the best that we can do
+        """
+    ''')
+    classic_mod = fromText('''
+    def f(a):
+        """
+        @param a: an arg, a the best of args
+        @type a: List[str]
+        @return: the best that we can do
+        @rtype: bool
+        """
+    ''')
+    def format(docstring):
+        stan = epydoc2stan.format_docstring(docstring)
+        stuff = []
+        flattenString(None, stan).addCallback(stuff.append)
+        return stuff[0].decode('ascii').replace('><', '>\n<')
```

Comment:
Most of this is already handled by `pydoctor.epydoc.markup.flatten()`, so using that could eliminate some duplicate code.

Reply:
Done